### DISCUSSION TITLE
Deploying in production

### DISCUSSION THREAD

**Discussion by pegesund** [rabbibotton/clog#306] [General]
Hi, any clues on the best way to move a server from debug mode to production? In debug mode I get all exceptions in the repl, for example..

Thanks for a very nice framework, development speed is fantastic :)

Petter

**Comment by aykaramba** [comment]
Sorry for dropping into this thread, I don't have an answer for you.  However I do have a question:

When you go into production, would you mind posting here what your config (roughly) is and how you are handling logging?

I would like to find out what best practice might be and document that.

**Reply by pegesund** [reply]
Will do!

**Comment by K1D77A** [comment]
I have a clog project that is deployed in a production environment. The steps for getting it setup are the same as any Common Lisp project.

**Reply by aykaramba** [reply]
Just dropping in to this thread again.  Outside of the rabbis webserver config below, did you do anything differently?  How are you handling CLOG loggin?

**Reply by K1D77A** [reply]
Put it all behind Nginx.

**Comment by rabbibotton** [comment]
https://github.com/rabbibotton/clog/blob/main/WEBSERVER.md may also be helpful
